jsettlers.mapcreator/src/jsettlers/mapcreator/control/IPlayerSetter.java

```diff
@@ -12,8 +12,18 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
  *******************************************************************************/
-package jsettlers.mapcreator.main;
+package jsettlers.mapcreator.control;
 
+/**
+ * Interface to get current active player
+ * 
+ * @author Andreas Butti
+ *
+ */
 public interface IPlayerSetter {
-	byte getActivePlayer();
+
+	/**
+	 * @return Active player
+	 */
+	public int getActivePlayer();
```

Comment:
* Methods defined in interfaces are always (implicitly) public methods => no need to write it down. 
* The game logik uses byte values for the player ids. If you'd use byte here, you cannot even get the problem of having a player with id 159. Although, I know the byte casts are annoying sometimes.